public/javascripts/AddInventory.js

```diff
@@ -216,17 +220,19 @@ var addInventory = {
 
         $("#pkg_add_button").prop("disabled", true);
 
-        navigation.hit("/AddProductSize/" + productId + "/" + name + "/" + size, function(res) {
-            $("#response").text("Added new package type: " + name + " " + size + ".");
-            addInventory.updatePackageTypes();
-            $("#pkg_name").val("");
-            $("#pkg_size").val("");
-            $("#pkg_add_button").prop("disabled", false);
-        })
-            /*.fail(function(res) {
-            $("#response").text("Error: Submit new package type: Connection error.");
-            $("#pkg_add_button").prop("disabled", false);
-        })*/;
+        navigation.get(window.apiRoute + "/AddProductSize/" + productId + "/" + name + "/" + size, function(err, res) {
+            if(err){
+                $("#response").text("Error: Submit new package type: Connection error.");
+                $("#pkg_add_button").prop("disabled", false);
+            }
+            else {
+                $("#response").text("Added new package type: " + name + " " + size + ".");
+                addInventory.updatePackageTypes();
+                $("#pkg_name").val("");
+                $("#pkg_size").val("");
+                $("#pkg_add_button").prop("disabled", false);
```

Comment:
No. It's just that the line is in both the if & else so it can go outside the if-else because it's the case for both.